File: examples/server/routes/languages/go/GET/yon.go
package main

import "strconv"

func Handler(request map[string]any) any {
	statusResponses := map[string]map[string]any{
		"411": {"code": "411", "detail": "length required"},
		"412": {"code": "412", "detail": "precondition failed"},
		"413": {"code": "413", "detail": "content too large"},
		"414": {"code": "414", "detail": "uri too long"},
		"415": {"code": "415", "detail": "unsupported media"},
	}
	if query, ok := request["query"].(map[string]any); ok {
		code := ""
		switch raw := query["code"].(type) {
		case string:
			code = raw
		case float64:
			code = strconv.Itoa(int(raw))
		}
		if response, ok := statusResponses[code]; ok {
			return response
		}
	}

	return GoLanguageService{}.Describe(request)
}

File: examples/server/routes/languages/csharp/GET/yon.cs
using System.Collections.Generic;
using System.Text.Json;

public class Yon
{
    private readonly CSharpLanguageService cSharpLanguageService = new CSharpLanguageService();

    public Dictionary<string, object?> Handler(JsonElement request)
    {
        var responses = new Dictionary<string, Dictionary<string, object?>>
        {
            ["423"] = Response("423", "locked"),
            ["424"] = Response("424", "failed dependency"),
            ["425"] = Response("425", "too early"),
            ["426"] = Response("426", "upgrade required"),
            ["428"] = Response("428", "precondition required"),
        };
        if (request.TryGetProperty("query", out var query)
            && query.TryGetProperty("code", out var raw))
        {
            var code = raw.ValueKind == JsonValueKind.Number
                ? raw.GetInt32().ToString()
                : raw.GetString() ?? "";
            if (responses.TryGetValue(code, out var response))
                return response;
        }

        return this.cSharpLanguageService.Describe(request);
    }

    private static Dictionary<string, object?> Response(string code, string detail)
    {
        return new Dictionary<string, object?>
        {
            ["code"] = code,
            ["detail"] = detail,
        };
    }
}
